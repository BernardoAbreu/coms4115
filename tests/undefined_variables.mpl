a = 6;
b = 7;

c = a + b;

if( d == true ) {
	c = c - 10;
}