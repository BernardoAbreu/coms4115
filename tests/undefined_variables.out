Fatal error: exception Failure("global d not found")
